libs/opmapcontrol/src/internals/core.cpp

```diff
@@ -65,6 +66,9 @@ namespace internals {
     Core::~Core()
     {
         ProcessLoadTaskCallback.waitForDone();
+        Matrix.Clear();
+        if (projection)
```

Comment:
deleting a null-pointer is a nop. So I would prefer something like this:
`delete projection;
projection = 0;`
Another possibility would be using an autoPointer.